VirtusLab-Open-Source/strapi-plugin-navigation
# feat: navigation internationalization [CU-29pdxpx]

## Note

This branch is based on `feat/typescript`

## Summary

- i18n capabilities for navigations
- new bootstrapping strategies
- TS typings updates

## Test Plan

- start application (expected result: everything stays the same)
- set default system language
- add other locales (like: `fr` or `pl`)
- add content types that use internationalisation
- enable i18n
  - new navigation versions are created
  - on navigation editing one can pick locale version
  - while editing locale version content types that are i18n and not at current locale should not be selectable
  - content without i18n should work as it used to
  - navigations should work as they are described in docs
  - graphQL schema should be aware of locale setting
- disable i18n
  - if navigations are not selected for pruning, they are still available
  - navigations are not grouped by language
